src/xdocs/config_naming.xml

```diff
@@ -2001,6 +2001,17 @@ class MyClass {
         </p>
         <source>
 &lt;module name=&quot;MethodName&quot;/&gt;
+        </source>
+        <p>Code Example:</p>
+        <source>
+class MyClass {
+  public void firstMethod1() {} // OK
+  protected void secondMethod2() {} // OK
+  private void ThirdMethod3() {} // violation, name 'ThirdMethod' must match
+                                 // pattern '^[a-z][a-zA-Z0-9]*$'
```

Comment:
Please change: 
`name 'ThirdMethod' must match
                                 // pattern '^[a-z][a-zA-Z0-9]*$'` to
`method name must match to default pattern "^[a-z][a-zA-Z0-9]*$"`

Reply:
Resolved